docs/source/pillars/information_governance.md

```diff
@@ -2,137 +2,260 @@
+* - You must gather and monitor the information governance requirements needed to fulfil the legal and ethical standards to protect data subjects and deliver valuable research.
+  - Requirements will come from a variety of sources including legislation, contractual obligations and ethical standards.
+    Requirements must be monitored to ensure the TRE controls continue to match such requirements.
+    New types of analysis or sources of data being brought into the scope of the TRE should be monitored to ensure requirements are current.
   - Mandatory
-* - You must have a process in place to monitor changes to any legal, ethical and contractual requirements, and to update your policies accordingly.
-  -
+```
+
+**Controls:** Measures, safeguards, or mechanisms implemented to manage or mitigate risks and ensure the integrity, confidentiality, availability, and reliability of systems, processes, or data.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-controls
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must ensure controls are implemented to ensure the requirements are met.
+  - Control implementation should be systematic and directly aligned to the internal and stakeholder requirements.
   - Mandatory
 ```
 
-## Quality management
+**Resource Allocation:** The process of assigning, distributing, and managing resources (such as personnel, finances, equipment, or time) within the TRE organisation to meet objectives and priorities effectively.
 
-_The ability of a TRE operator to measure and control quality of processes, documentation and outputs._
+```{list-table}
+:header-rows: 1
+:name: tab-ig-resource-allocation
 
-### Document management
+* - Statement
+  - Guidance
+  - Importance
+* - You must ensure there are adequate resources to provide assurance and meet information governance requirements.
+  - There should be adequate access to funding to implement and maintain the TRE which is under the direct control of the TRE organisation.
+  - Mandatory
+```
+
+## Quality Management
+
+What the organisation does to measure and control quality of processes, documentation and outputs.
+
+**Document and SOP Management:** Creating, organising, updating, and controlling documents and Standard Operating Procedures (SOPs) within the TRE organisation.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-document-management
+:name: tab-ig-quality-management
 
 * - Statement
   - Guidance
   - Importance
-* - You must control all of your policies and standard operating procedures.
-  - This may include measures like restricting edit access to relevant documents, and recording acceptance of policies for all TRE operators.
+* - You must ensure all policies and standard operating procedures relevant to the TRE organisation are controlled.
+  - This may include measures like restricting edit access to relevant documents and recording acceptance of policies for all TRE organisation members.
```

Comment:
Can we define TRE organisation members?

Reply:
Can you think of a generic phrase for everyone running/operating/governing/using the TRE?

Reply:
#160 has a definition of TRE organisation. Should we merge that as a starting point?

Reply:
OK, shall we work on the assumption that https://github.com/sa-tre/satre-specification/pull/160 will be merged soon?

Reply:
I'm happy to merge it now (without most of the definitions) but if anyone (@machintim?) has some definitions they want to add, let's add those to #160 today.